landscaper/pkg/controlplane/apis/exports/exports.go

```diff
@@ -16,20 +16,32 @@ package exports
 
 // Exports defines the structure for the exported data which might be consumed by other components.
 type Exports struct {
-	GardenerIdentity                         string `json:"gardenerIdentity" yaml:"gardenerIdentity"`
-	GardenerAPIServerEncryptionConfiguration string `json:"gardenerAPIServerEncryptionConfiguration" yaml:"gardenerAPIServerEncryptionConfiguration"`
-	OpenVPNDiffieHellmanKey                  string `json:"openVPNDiffieHellmanKey" yaml:"openVPNDiffieHellmanKey"`
+	// GardenerIdentity is the identity of the Gardener installation
+	GardenerIdentity string `json:"gardenerIdentity"`
+	// GardenerAPIServerEncryptionConfiguration is the encryption configuration of the Gardener API Server
+	GardenerAPIServerEncryptionConfiguration string `json:"gardenerAPIServerEncryptionConfiguration"`
+	// OpenVPNDiffieHellmanKey is the Diffie-Hellman-Key used for the Shoot<->Seed VPN
+	OpenVPNDiffieHellmanKey string `json:"openVPNDiffieHellmanKey"`
 
-	GardenerAPIServerCA           Certificate  `json:"gardenerAPIServerCA" yaml:"gardenerAPIServerCA"`
-	GardenerAdmissionControllerCA *Certificate `json:"gardenerAdmissionControllerCA,omitempty" yaml:"gardenerAdmissionControllerCA,omitempty"`
+	// GardenerAPIServerCA is the PEM encoded CA certificate of the Gardener API Server
+	GardenerAPIServerCA Certificate `json:"gardenerAPIServerCA" yaml:"gardenerAPIServerCA"`
```

Comment:
Why only this field has a yaml tags but others don't?

Reply:
Thanks, none of the needs the yaml tag. I removed it.